### PR TITLE
Don't overwrite last modified dates on sync.

### PR DESCRIPTION
**Pull request by nickmshelley**
Notify of changes once at end of sync.

### CODE REVIEW

**Comment by nickmshelley** [comment]
Why are github's diffs so horrible?

**Comment by nickmshelley** [comment]
Here's what the diff looks like in Xcode.
screen shot 2016-10-27 at 4 51 48 pm

**Comment by nickmshelley** [comment]
Tests passed locally.

**Comment by stephanheilner** [comment]
👍 

The pre-load existing annotations will be in a follow-on commit?

File: LDSAnnotations/AnnotationStore+Annotation.swift [hunk 1]
```diff
@@ -333,7 +333,10 @@ extension AnnotationStore {
                 AnnotationTable.device <- device
             ))
             
-            try self.notifyModifiedAnnotationsWithIDs([id], source: source)
+            // Sync notifies once at the end
+            if case .Local = source {
```
Comment (stephanheilner) [inline LDSAnnotations/AnnotationStore+Annotation.swift]: I've never seen this syntax before. Pretty slick!

**Comment by nickmshelley** [comment]
Yeah I figured it would be best to get the modified date bug fix in sooner rather than later.

**Pull request merged**